src/engines/pallet_engine/template.rs

```diff
@@ -4,23 +4,31 @@ use crate::{
 };
 use std::{fs, path::PathBuf};
 
+
 pub fn create_pallet_template(
 	path: Option<String>,
 	config: TemplatePalletConfig,
 ) -> anyhow::Result<()> {
 	let target = resolve_pallet_path(path);
+	// TODO : config.name might use `-` or use snake_case. We want to use pallet_template for the pallet dirs
```

Comment:
Why adding it?

Reply:
Don't think it's necessary anymore. Will remove.